CMakeLists.txt

```diff
@@ -478,6 +478,66 @@ if (OPENSSL_FOUND AND CUDAQ_ENABLE_REST)
   add_subdirectory(tpls/Crow)
 endif()
 
+# AWS SDK for C++
+set(AUTORUN_UNIT_TESTS OFF CACHE BOOL "" FORCE)
+set(AWS_SDK_WARNINGS_ARE_ERRORS OFF CACHE BOOL "" FORCE)
+set(AWS_USER_AGENT_CUSTOMIZATION "CUDA-Q/${CUDA_QUANTUM_VERSION}" CACHE STRING "" FORCE)
+set(BUILD_ONLY "braket;s3-crt;sts" CACHE STRING "" FORCE)
+set(BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
```

Comment:
```suggestion
```
Passed directly to execute process, and the configuration for the rest of the build should not be impacted.

Reply:
Fixed in 63d84bbbe956e72a2737b6cce1123ef301823002